File: Practica 10/Python/Rectangulo.py
from Figura import Figura

class Rectangulo(Figura):
    def __init__(self, color: str, width: float, height: float):
        super().__init__(color)
        self.width = width
        self.height = height
    
    def perimetro(self) -> float:
        return 2 * (self.width + self.height)
    
    def area(self) -> float:
        return self.width * self.height

File: Practica 10/Java/Rectangulo.java
package Java;

public class Rectangulo extends Figura {
    private double width, height;

    public Rectangulo(String color, double width, double height) {
        super(color);
        this.width = width;
        this.height = height;
    }

    @Override
    public double perimetro() {
        return 2 * (width + height);
    }

    @Override
    public double area() {
        return width * height;
    }
}